File: Year 2/Semester 4/Programming and Projection Enironments/Week 5/JavaFifthWeek/src/main/java/com/example/temasaptamana5/repository/interfaces/ITicketRepository.java
package com.example.temasaptamana5.repository.interfaces;

import com.example.temasaptamana5.domain.Entity;
import com.example.temasaptamana5.domain.Ticket;
import com.example.temasaptamana5.repository.IRepository;

public interface ITicketRepository <ID, E extends Entity<ID>> extends IRepository<ID, E> {
    void addTicket(Ticket ticket);
}

File: Year 2/Semester 4/Programming and Projection Environments/Week 6/ASaseaSaptamana/ASaseaSaptamana/repository/interfaces/ITicketRepository.cs
﻿using ASaseaSaptamana.domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ASaseaSaptamana.repository.interfaces
{
    public interface ITicketRepository : IRepository<int, Ticket<int>>
    {
        void AddTicket(Ticket<int> ticket);
    }
}